test/test_cluster.js

```diff
@@ -1742,4 +1742,493 @@ describe('Cluster', function () {
+            .auth(common.credentials.user, common.credentials.password)
+            .expect("Content-type",/json/)
+            .expect(200)
+            .end(function(err,res){
+                if (err) return done(err);
+
+                res.body.should.have.properties(['error', 'data']);
+                res.body.data.should.have.properties(['rootcheck']);
+                res.body.data.rootcheck.should.have.properties(['check_unixaudit', 'check_sys', 'rootkit_trojans',
+                'skip_nfs', 'check_if', 'check_pids', 'check_dev', 'check_ports', 'disabled', 'rootkit_files',
+                // 'frequency', 'scanall', 'check_trojans', 'base_directory', 'check_files', 'system_audit']); // base directory value is empty, this cause an error
+                'frequency', 'scanall', 'check_trojans', 'check_files', 'system_audit']);
+
+                res.body.error.should.equal(0);
+                done();
+            });
+        });
+
+        it('Request-Syscheck-Internal', function(done) {
+            request(common.url)
+            .get("/cluster/master/config/syscheck/internal")
+            .auth(common.credentials.user, common.credentials.password)
+            .expect("Content-type",/json/)
+            .expect(200)
+            .end(function(err,res){
+                if (err) return done(err);
+
+                res.body.should.have.properties(['error', 'data']);
+                res.body.data.should.have.properties(['internal']);
+                res.body.data.internal.should.have.properties(['syscheck', 'rootcheck']);
+
+                res.body.error.should.equal(0);
+                done();
+            });
+        });
+
+        // wmodules
+		it('Request-Wmodules-Wmodules', function(done) {
+            request(common.url)
+            .get("/cluster/master/config/wmodules/wmodules")
+            .auth(common.credentials.user, common.credentials.password)
+            .expect("Content-type",/json/)
+            .expect(200)
+            .end(function(err,res){
+                if (err) return done(err);
+
+                res.body.should.have.properties(['error', 'data']);
+                res.body.data.should.have.properties(['wmodules']);
+                //res.body.data.wmodules.should.have.properties(['open-scap', 'cis-cat',
+                //'osquery', 'syscollector', 'database', 'wazuh_download']);
+                res.body.data.wmodules[0].should.have.properties(['open-scap']);
+                res.body.data.wmodules[1].should.have.properties(['syscollector']);
+                res.body.data.wmodules[2].should.have.properties(['vulnerability-detector']);
+                res.body.data.wmodules[3].should.have.properties(['cis-cat']);
+                res.body.data.wmodules[4].should.have.properties(['database']);
+                res.body.data.wmodules[5].should.have.properties(['wazuh_download']);
+
+
+                res.body.error.should.equal(0);
+                done();
+            });
+        });
+
+
```

Comment:
I miss some SCA module tests.

Reply:
Fixed in https://github.com/wazuh/wazuh-api/pull/362/commits/f9d53bf3ab682d8b8ae30538f21b65990fe96c42